plugins/inputs/logfile/logfile_test.go

```diff
@@ -358,100 +358,128 @@ func TestLogsFileRemove(t *testing.T) {
 	tt.Stop()
 }
 
-//When another file is created for the same file config and the file config has auto_removal as true, the old files will stop at EOF and removed afterwards
-func TestLogsFileAutoRemoval(t *testing.T) {
```

Comment:
This old test seems to create a temp file and write 1 line to it.
Then spawn a goroutine to create a 2nd temp file.
However, the test read the 1 line from the  1st temp file before the goroutine runs and create the 2nd file, so it was not an effective test.